families/identity/protos/setting.proto

```diff
@@ -0,0 +1 @@
+../../../protos/setting.proto
```

Comment:
Are these necessary? Could the path to the external proto file just be hard coded in build.rs?